index.bs

```diff
@@ -6794,12 +6794,13 @@ This [=client extension|client=] [=registration extension=] facilitates reportin
     </xmp>
 
 : Client extension processing
-::  None, other than to report on credential properties in the output.
+::  [=map/Set=] <code>[=credentialCreationData/clientExtensionResults=]["{{AuthenticationExtensionsClientOutputs/credProps}}"]["rk"]</code>
+    to the value of the |requireResidentKey| parameter that was used in the
+    <a href='#CreateCred-InvokeAuthnrMakeCred'>invocation</a> of the [=authenticatorMakeCredential=]
+    operation.
```

Comment:
I think these steps should also include setting `authenticatorDisplayName`? See meta-PR #1995 for that and another bit of polish I noticed.

Reply:
Thank you for helping flesh out the client processing rules, I've merged your PR into this one! 🙏